griptape/drivers/image_query_model/bedrock_claude_image_query_model_driver.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+from typing import Optional
+from attr import field, define
+from griptape.artifacts import ImageArtifact, TextArtifact
+from griptape.drivers import BaseImageQueryModelDriver
+
+
+@define
+class BedrockClaudeImageQueryModelDriver(BaseImageQueryModelDriver):
+    """
+    Attributes:
+        max_output_tokens: Max output tokens to return.
+    """
+
+    max_output_tokens: Optional[int] = field(default=4096, kw_only=True, metadata={"serializable": True})
+
+    def construct_image_query_request_parameters(self, query: str, images: list[ImageArtifact]) -> dict:
+        content = []
+        for image in images:
+            content.append(self._construct_image_message(image))
+
+        content.append(self._construct_text_message(query))
+        messages = self._construct_messages(content)
+
+        input_params = {"messages": messages, "anthropic_version": "bedrock-2023-05-31"}
```

Comment:
Let's make `anthropic_version` an field variable for consistency with `AnthropicPromptDriver`

Reply:
Ah sorry I meant this guy: https://github.com/griptape-ai/griptape/blob/dev/griptape/drivers/prompt_model/bedrock_claude_prompt_model_driver.py#L17.

Agreed that it probably won't change on us, but I think it should be moved to at least a static class field since it's kind of a magic string. Maybe we don't need to expose it to the user via an instance field.

Reply:
I made it a static field and updated `BedrockClaudePromptModelDriver` as well to stay consistent.